src/Frontend/Page/User.hs

```diff
@@ -113,11 +113,11 @@ userSettings = FormPageHandler (PageUserSettings <$> currentUser) changeUser
         updatePass uid oldPass newPass1 newPass2
         pure ()
 
-    updateEmail uid em = do
+    updateEmail uid em = do  -- TODO: if email hasn't changed, do not add message.
```

Comment:
I suggest that `setUserEmail` in `Persist` returns whether or not the email has changed. We could have a variation of `.~` and `?~`.

However I'm also wondering if the lack of message be concerning, a single message saying that the changes are now saved might be enough.